src/test/java/teammates/test/cases/ui/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -243,18 +219,14 @@ public void testExecuteAndPostProcess() throws Exception {
         };
         
         a = getAction(params);
-        rr = (RedirectResult) a.executeAndPostProcess();
+        ajaxResult = (AjaxResult) a.executeAndPostProcess();
+        editCopyData = (InstructorFeedbackEditCopyData) ajaxResult.data;
         
-        expectedString = Const.ActionURIs.INSTRUCTOR_FEEDBACKS_PAGE
-                         + "?error=true"
-                         + "&user=" + instructor.googleId
-                         + "&courseid=" + instructor.courseId
-                         + "&fsname=First+Session";
-        assertEquals(expectedString, rr.getDestinationWithParams());
+        assertEquals("", editCopyData.redirectUrl);
```

Comment:
Stick to one of these for all of the tests:
`assertTrue(editCopyData.redirectUrl.isEmpty());`
`assertEquals("", editCopyData.redirectUrl);`
I prefer the first, but both convey the meaning well enough.